app/domain/content/importers/content_details.rb

```diff
@@ -0,0 +1,17 @@
+module Content
```

Comment:
We don't need to scope any more, as we do not have the Audit Tool has been moved out of the CPM